tests/bdd/scale/test_NAS_T1062.py

```diff
@@ -111,14 +114,21 @@ def click_the_start_automatically_ssh_checkbox_and_enable_the_ssh_service(driver
     value_exist = attribute_value_exist(driver, '//mat-slide-toggle[@ix-auto="slider__state__SSH"]', 'class', 'mat-checked')
     if not value_exist:
         driver.find_element_by_xpath('//div[@ix-auto="overlay__stateSSH"]').click()
-    time.sleep(1)
+    time.sleep(4)
 
 
 @then('the service should be enabled with no errors')
 def the_service_should_be_enabled_with_no_errors(driver):
     """the service should be enabled with no errors."""
-    wait_for_value = wait_for_attribute_value(driver, 7, '//mat-slide-toggle[@ix-auto="slider__state__SSH"]', 'class', 'mat-checked')
-    assert wait_for_value
+    assert wait_on_element(driver, 10, '//mat-list-item[@ix-auto="option__Dashboard"]', 'clickable')
```

Comment:
Why does this go to the Dashboard?
you need to rework the section.

You do not need to go to Dashboard and come back.

Reply:
oh, that was a temp fix because the page was screwed up with UI updates because the slider thing wouldnt update on the page unless you reloaded it. I'll pull that out.